torch/distributed/fsdp/_runtime_utils.py

```diff
@@ -906,6 +894,22 @@ def _post_backward_hook(
                         handle.flat_param._cpu_grad = None
 
 
+@no_type_check
+def _log_post_backward_hook(state: _FSDPState, handle: FlatParamHandle) -> None:
+    # Under TORCH_DISTRIBUTED_DEBUG=INFO, log the module names this hook fires for.
+    # Below logging of module names this post-bwd hook fires for can help debug certain
+    # cases where hooks don't fire, such as under certain activation checkpoint configs.
+    if state._use_orig_params and handle._debug_level == dist.DebugLevel.INFO:
+        param_to_fqn = state._exec_order_data.param_to_fqn
+        handle_params = handle.flat_param._params  # only populated for use_orig_params
+        param_fqns = [
+            param
+            for param_list in [param_to_fqn[p] for p in handle_params]
+            for param in param_list
+        ]
+        log.warning("FSDP firing post-backward hooks for parameters %s", param_fqns)
```

Comment:
Curious why this is warning but line 902 is INFO?

Reply:
I guess the question is whether we expect tight coupling between `TORCH_DISTRIBUTED_DEBUG=INFO` and `log.info` and between `TORCH_DISTRIBUTED_DEBUG=DETAIL` and `log.warning`.

cc @rohan-varma  who added this